src/burnchains/burnchain.rs

```diff
@@ -504,13 +504,22 @@ impl Burnchain {
         self.first_block_height + reward_cycle * (self.pox_constants.reward_cycle_length as u64) + 1
     }
 
-    pub fn block_height_to_reward_cycle(&self, block_height: u64) -> Option<u64> {
-        if block_height < self.first_block_height {
+    pub fn static_block_height_to_reward_cycle(
```

Comment:
function comment.. in particular, what is `first_block_ht` in relation to `block_ht`?

Reply:
`first_block_ht` == first burnchain block height at which Stacks blocks are mined
`block_ht` == current burnchain block height

Reply:
Added in bbd826d4bde81aee6c26f134439b9d73d8885e34